bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java

```diff
@@ -0,0 +1,82 @@
+package de.agilecoders.wicket.core.markup.html.bootstrap.button.dropdown;
+
+import org.apache.wicket.Component;
+import org.apache.wicket.markup.ComponentTag;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.apache.wicket.util.lang.Args;
+
+import at.lerchen.wicket.bootstrap4.DropDownAlignmentBehavior.Alignment;
+import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.BootstrapBaseBehavior;
+import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.ICssClassNameProvider;
+import de.agilecoders.wicket.core.util.Attributes;
+
+public class DropDownVariationBehavior extends BootstrapBaseBehavior {
```

Comment:
Please add a javadoc what this behavior is good for and when it should be used.

Reply:
done